### PR TITLE
Added first version of Illuminate Mail

### PR DESCRIPTION
**Pull request by Nyholm**
Related to #547

### CODE REVIEW

**Comment by Nyholm** [comment]
I dont really get the Psalm issues. Could I get some help?

File: src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    {
+        $this->beforeSendPerformed($message);
+
+        $input = [
+            'Content' => new EmailContent([
+                'Raw' => ['Data' => $message->toString()],
+            ]),
+        ];
+
+        $from = key($message->getSender() ?: $message->getFrom());
```
Comment (jderusse) [inline src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php:55]: getFrom returns mixed, `key()` expects an array
https://github.com/swiftmailer/swiftmailer/blob/4633950338d22b4406e5c77f6e25c658435d5db4/lib/classes/Swift/Mime/SimpleMessage.php#L223-L225

Comment (Nyholm) [reply src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php:56]: Oh. That wasn't the psalm error I was fightning.. Hm.. Or maybe it was but the message was different.

Comment (Nyholm) [reply src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php]: Could you help with this?
https://github.com/async-aws/aws/pull/550/checks?check_run_id=652813243#step:5:10

Comment (jderusse) [reply src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php]: .. I don't see the issue

Comment (Nyholm) [reply src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php]:
```
Note: Using configuration file /github/workspace/phpstan.neon.dist.
 ------ -------------------------------------------------------------------- 
  Line   Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php     
 ------ -------------------------------------------------------------------- 
  51     Parameter #1 $input of class AsyncAws\Ses\ValueObject\EmailContent  
         constructor expects array()|array('Simple' =>                       
         array|AsyncAws\Ses\ValueObject\Message|null, 'Raw' =>               
         array|AsyncAws\Ses\ValueObject\RawMessage|null, 'Template' =>       
         array|AsyncAws\Ses\ValueObject\Template|null), array('Raw' =>       
         AsyncAws\Ses\ValueObject\RawMessage) given.                         
 ------ -------------------------------------------------------------------- 
```

File: phpstan-baseline.neon [hunk 2]
```diff
@@ -1,2 +1,7 @@
 parameters:
-    ignoreErrors:
+	ignoreErrors:
+		-
+			message: "#^Parameter \\#1 \\$input of class AsyncAws\\\\Ses\\\\ValueObject\\\\EmailContent constructor expects array\\(\\)\\|array\\('Simple' \\=\\> array\\|AsyncAws\\\\Ses\\\\ValueObject\\\\Message\\|null, 'Raw' \\=\\> array\\|AsyncAws\\\\Ses\\\\ValueObject\\\\RawMessage\\|null, 'Template' \\=\\> array\\|AsyncAws\\\\Ses\\\\ValueObject\\\\Template\\|null\\), array\\('Raw' \\=\\> AsyncAws\\\\Ses\\\\ValueObject\\\\RawMessage\\) given\\.$#"
+			count: 1
+			path: src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php
```
Comment (Nyholm) [inline phpstan-baseline.neon:6]: I cannot reproduce this error here: https://phpstan.org/r/e109b47f-d7e5-482f-920d-e659b0d945e7

**Pull request merged**